Hw.js

```diff
@@ -0,0 +1,19 @@
+
+//Multiplication Tables
+var multiply = 9;
+for (var i=0; i<=10; i++ ){
+	console.log("2 * 9 = "+ i * multiply );
+}
```

Comment:
Almost! This will always read `2 * 9 = xxx` we want it to say `x * 9 = xxx`. Good job though!